doc/tags/for.rst

```diff
@@ -85,13 +85,14 @@ Adding a condition
 ------------------
 
 Unlike in PHP, it's not possible to ``break`` or ``continue`` in a loop. You
-can however filter the sequence during iteration which allows you to skip
-items. The following example skips all the users which are not active:
+can however apply a :doc:`filter<filters/filter>` filter onto the sequence
+which allows you, effectively, to skip items. The following example skips all
+the users which are not active:
 
 .. code-block:: twig
 
     <ul>
-        {% for user in users if user.active %}
+        {% for user in users|filter(v => v.active) %}
```

Comment:
I would use `user` or `u` for the variable.

Reply:
You should also add a note to say that `filter` is available as of Twig 1.41/2.10 (and mention the old syntax with the `if` condition).